src/utils/Validator/ValidatorWithMessage.ts

```diff
@@ -0,0 +1,39 @@
+import ValidationRule from "./ValidationRule";
+import {UIInputEvent, ValidationResult, ValidResult} from "./types";
+import ValidatorI from "./ValidatorI";
+
+const defaultRules = {
+    input: {
+        default: [new ValidationRule(()=> true, '')]
+    },
+    blur: {
+        default: [new ValidationRule(()=> true, '')]
+    }
+}
+
+class ValidatorWithMessage implements ValidatorI {
```

Comment:
Having both the `Validator` class and the `ValidatorWithMessage` class is good to comparing and testing different scenarios but before merging it would be better to remove one and stay with a class which could handle both scenarios.

Reply:
Hi Pablo,

The `Validator` class is still in use. Am not too sure if you are referring to merging the classes that use `Validator` to use `ValidatorWithMessage`?